Makefile.base

```diff
@@ -62,6 +62,10 @@ ASSMOBJ     := $(ASSMSRC:%.S=$(BINDIR)/$(MODULE)/%.o)
 OBJ := $(OBJC) $(OBJCXX) $(ASMOBJ) $(ASSMOBJ)
 DEP := $(OBJC:.o=.d) $(OBJCXX:.o=.d) $(ASSMOBJ:.o=.d)
 
+ifneq (,$(filter micropython,$(USEPKG)))
+  include $(RIOTMAKE)/py.inc.mk
```

Comment:
can this be moved to one of the Makefiles in `pkg/micropython/` ?

I'm not sure this is already answered, but generally it looks weird to have (single) package specific handling in here

Reply:
I've factored this out into a generically usable and independently tested blob header: #11497.